### PR TITLE
Add notes from June 2024

### PR DESCRIPTION
**Pull request by hugovk**
----
📚 Documentation preview 📚: https://docs-community--117.org.readthedocs.build/en/117/monthly-meeting/2024-06.html

### CODE REVIEW

**Review by willingc — APPROVED** [review]
Thank you @hugovk 🎉

File: docs/monthly-meeting/2024-06.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        - Open an issue to ask for a decision
+        - Moving docs from Google Drive to the repo for visibility
+    - If folks get blocked, opening an issue on the EB repo is a good way to get in touch.
+    - The devguide serves the core dev community more than the general docs community.
+      We'll want to put guidelines in the docs repo instead. We'll move slow enough to not disrupt the devguide.
+      We'll make it accessible to folks who want to contribute more frequently.
+    - [Hugo] Yes, the first step in the devguide is to build CPython. Not ideal
+    - [Carol] We had folks like Melanie and Shauna go through the guide and take notes.
+      We're planning to put the guide on docsguide.python.org; open it up.
+    - Feel free to open issues on the EB repo to give Ned notes & feedback.
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2024-06.md:72]:
```suggestion
- [Carol]
  - PyCon Highlights
    - Dinner with cross-section of core devs, educators, and authors
    - Sprint contribute to docs feedback
    - Sprint discussions
  - [Draft Guiding Principles for Documentation](https://drive.google.com/file/d/1elKNdRPTIoNe6EZ4TbS9aquM-RaNDgmc/view)
  - [Draft meeting notes from yesterday](https://docs.google.com/document/d/1NkaA4LMltE_GISN5v52bWfxA_QGcRxw6U3jzwFQ7c_c/edit?usp=sharing)
  - [Editorial Board Repo](https://github.com/python/editorial-board) (share feedback here)
    - Open an issue to ask for a decision
    - Moving docs from Google Drive to the repo for visibility
  - If folks get blocked, opening an issue on the EB repo is a good way to get in touch.
  - The devguide serves the core dev community more than the general docs community.
    We'll want to put guidelines in the docs repo instead. We'll move slow enough to not disrupt the devguide.
    We'll make it accessible to folks who want to contribute more frequently.
  - [Hugo] Yes, the first step in the devguide is to build CPython. Not ideal
  - [Carol] We had folks like Melanie and Shauna go through the guide and take notes.
    We're planning to put the guide on docsguide.python.org; open it up.
  - Feel free to open issues on the EB repo to give Ned notes & feedback.
```

File: docs/monthly-meeting/2024-06.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+* [Marcus] From an instructional standpoint,
+  the EB guiding principles are very comforting.
+  People jumping into programming from e.g. Excel struggle with e.g. `logging`.
+  It's great that we're now focusing more on editorial issues instead of just
+  the infrastructure.
+
+* [Ryan] Should we put examples in docstrings? They are very helpful
+  - [Hugo] We've historically intentionally kept docstrings shorter.
+    The Python documentation is duplicated --
+    both in the docstrings and the main docs.
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2024-06.md:136]:
```suggestion
  - [Hugo] We've historically intentionally kept docstrings shorter.
    The Python documentation is duplicated --
    both in the docstrings and the main docs.
  - [Ezio] Docstrings should just serve as quick reminders,
    they are not supposed to be exhaustive.
```
(I mentioned this in the chat.)

**Review by ezio-melotti — COMMENTED** [review]
I would keep the fixes to the other notes in a separate PR just to keep the history clean, but it's not a big deal.

**Pull request merged**